lms/djangoapps/lti_provider/views.py

```diff
@@ -58,8 +61,18 @@ def lti_launch(request, course_id, usage_id):
     # Check the LTI parameters, and return 400 if any required parameters are
```

Comment:
The statement above:

```
if not settings.FEATURES['ENABLE_LTI_PROVIDER']:
    return HttpResponseForbidden()
```

As I understand you should replace this string with your custom template too ?